pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2283,3 +2283,13 @@ def test_merge_join_categorical_multiindex():
     expected = expected.drop(["Cat", "Int"], axis=1)
     result = a.join(b, on=["Cat1", "Int1"])
     tm.assert_frame_equal(expected, result)
+
+
+@pytest.mark.parametrize("how", ["left", "right"])
```

Comment:
sort=True is ok? (do we test this)? can you locate this test function near similar tests.

Reply:
Sort True was the previous default behavior. Added tests to show behavior. Moved the tests up a bit right below a nosort test